ubuntu/14.04/standard/Dockerfile

```diff
@@ -132,6 +145,11 @@ RUN apt-get update \
     python3-pip \
  && rm -rf /var/lib/apt/lists/*
 
+# Install Scala build tools
```

Comment:
This doesn't look like an official version: https://github.com/paulp/sbt-extras, the license also has some requirements for distribution, we will need to check on that

Reply:
It is not an official version, however sbt-extras allows for downloading the version of Scala and sbt required by a project. @DavidStaheli is the licensing a concern?